content/docs/datamapper/01-getting-started/_index.md

```
@@ -0,0 +1,100 @@
+Follow the below steps to add a parameter.
+
+1. Click the plus **+** button on the right side of the `Parameters` title.
+{{< image-sh src="datamapper-add-parameter.png" text="Parameters" >}}
+
+2. Now type the parameter name and click the check button on the right.
+{{< image-sh src="datamapper-add-parameter-confirm.png" text="Add Parameter confirm" >}}
+
+> [!TIP]
+> When you have many parameters, you can improve readability by hiding them visually. Click the eye icon next to the Parameters section to toggle the visibility of the parameter list.
+> {{< image-sh src="datamapper-hide-parameters.png" text="Hide parameters to improve readability" >}}
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)